lib/Parser.js

```diff
@@ -1097,14 +1092,11 @@ class Parser extends Tapable {
 	}
 
 	walkExpression(expression) {
-		if(this["walk" + expression.type])
-			return this["walk" + expression.type](expression);
+		this._walkNode(expression);
 	}
 
 	walkAwaitExpression(expression) {
-		const argument = expression.argument;
-		if(this["walk" + argument.type])
-			return this["walk" + argument.type](argument);
+		this._walkNode(expression.argument);
```

Comment:
better call `this.walkExpression(expression.argument)` here. That makes more sense.